### DISCUSSION TITLE
Reusing a saved template

### DISCUSSION THREAD

**Discussion by guydvir2** [arendst/Tasmota#11325] [Q&A] [answered]
Hi,
I created a windows blinds roller, mostly using the on-line docs, on a Sonoff Dual. After all worked OK, I saved it to a template, and tried to re-use it on another Node-MCU.
Template was uploaded to Node-MCU, MQTT topic was changed, Hostname, Rules deleted and even turned OFF, but when operating NodeMCU via WEBUI it also affected to one the was installed on a real Window ( when Sonoff Dual ).

what might it be ??
PS. even Device Group, and group topics were deleted...

**Comment by barbudor** [comment, marked as answer]
If operating one device interact with another it's likely that you didn't change correctly the topic or devicegroup

Enable Weblog 3 on each device, interact with one and check the logs on the other one to understand where the command is coming from. It should display if the source is MQTT (topic) or DeviceGroup

**Reply by barbudor** [reply]
Beware that an DevGroupName of `""` is a valid DevGroupName. So if DevGroup are enabled by `SetOption85 1`, all devices with enabled DebGroup and one of theire group is `""` will react.
So if you need to leave DevGroup enabled but don't wan't to share, be sure to define a dummy individual DevGroupName1..4 on each device.

**Reply by guydvir2** [reply]
Well, you are right again:

`12:28:51.641 CMD: DevGroupName
12:28:51.643 SRC: WebConsole from 192.168.3.49
12:28:51.645 CMD: Group 0, Index 1, Command "DEVGROUPNAME", Data ""
12:28:51.649 RSL: RESULT = {"DevGroupName1":"","DevGroupName2":"","DevGroupName3":"","DevGroupName4":""}`

BUT 
`SetOption85` was `1` on both devices.... so only after `SetOption85 0' on both, solved it.
Thanks, again.
Guy
